modules/mqtt/mqtt-grammar.ym

```diff
@@ -66,6 +66,8 @@ MQTTClientOptions *last_options;
 %token KW_CERT_FILE
 %token KW_KEY_FILE
 %token KW_CIPHER_SUITE
+%token KW_TLS12_AND_OLDER
+%token KW_TLS13
```

Comment:
I think that adding this keyword might be misleading because  (as you have mentioned in the commit message) 
> tls13() is currently not supported by paho-mqtt-c.

Currently if the user sets it the error message is the following:
> Error parsing mqtt, syntax error, unexpected KW_TLS13, expecting ')' in [...]

Reply:
Yep, I just declared the keyword without using it in non-terminal rules, so we can just add a rule when paho-mqtt-c adds support for restricting TLS 1.3 ciphers. I can remove it, if you think it's better.

Reply:
Please do so, as currently `config_option_database` will advertise it if it's here:

`/syslog-ng-cfg-db.py -c destination -d mqtt`

```
[...]
tls(
    <empty>
    ca-dir(<string>)
    ca-file/cacert(<path>)
    cert/cert-file(<path>)
    key-file(<path>)
    peer-verify(<yesno>)
    ssl-version(<string>)
    use-system-cert-store(<yesno>)
    cipher-suite(
      <empty>
      <string>
      tls12-and-older(<string>)
      tls13(<string>)
    )
  )

```

Reply:
This must be a bug in the cfg-db script, this option is not part of the grammar rules of MQTT.

Reply:
Yes it is a bug in the script. Hopefully it will be fixed with the next refactor.